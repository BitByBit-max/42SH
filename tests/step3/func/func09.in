say_hello()
{
	echo "hello $1!!";
	( echo "How are you?" && echo "are you okay ?" ) | grep "How";
} > in

say_hello "Elise"

cat in

rm in
