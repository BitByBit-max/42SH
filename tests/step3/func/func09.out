hello Elise!!
How are you?
